<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-A9SHFLG8" name="Using_SoftAssert" tests="3" failures="1" timestamp="15 Sep 2017 21:54:24 GMT" time="0.019" errors="0">
  <testcase name="LoginTest" time="0.003" classname="Testcases.Using_SoftAssert">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error message 1 ,
	expected [B] but found [A]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	error message 1 ,
	expected [B] but found [A]
	at Testcases.Using_SoftAssert.LoginTest(Using_SoftAssert.java:77)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase name="PassWordChangeTest" time="0.0" classname="Testcases.Using_SoftAssert">
    <skipped/>
  </testcase> <!-- PassWordChangeTest -->
  <testcase name="LogoutTest" time="0.0" classname="Testcases.Using_SoftAssert">
    <skipped/>
  </testcase> <!-- LogoutTest -->
</testsuite> <!-- Using_SoftAssert -->
